src/sentry/api/endpoints/organization_projects_experiment.py

```diff
@@ -89,6 +90,10 @@ def post(self, request: Request, organization: Organization) -> Response:
 
         if not features.has("organizations:team-roles", organization):
             raise ResourceDoesNotExist(detail=MISSING_PERMISSION_ERROR_STRING)
+        if organization.flags.disable_member_project_creation and not request.access.has_scope(
+            "org:write"
+        ):
```

Comment:
Still learning the models, so sorry for the basic questions:
- Will admins always have the `org:write` scope attached to their `access`?
- Is the permission/scope limited to the project, or is it a global user permission?
- What about if this is invoked through an API token?
- Is seems like based on `OrgProjectPermission`, the `org:write` is already required, and the endpoint will fail if that scope is not part of the request, so is this line still needed?

Reply:
> Is seems like based on OrgProjectPermission, the org:write is already required, and the endpoint will fail if that scope is not part of the request, so is this line still needed?

Nevermind, we overwrite the `POST` permissions, and the source map is not additive, so this line is needed

Reply:
org admins have `org:write`, but regular org members can become team admins, in which case they don't have `org:write. Org admins are automatically made into team admins when they join teams.

Reply:
Scopes are attached to each user depending on their role in the organization. For example, org owners are the only ones with `org:admin`.

Api tokens have scopes attached to them, which you can select when making a user token or integration token

Reply:
Would a regular member/user have access to tokens that have this elevated permission (scope)? Trying to understand if a user can side step this permission check even if it's not coming directly from them. So even if I'm blocked on the UI, I could get around this if I go through automation

Reply:
User tokens can have any scopes b/c it can be applied to multiple orgs the user is part of. We take the intersection between the user token's scopes and the user's role in the org.
Integration tokens are tied to orgs, but we don't allow regular members to create them